patches/amd-mainline/rocBLAS/0004-Run-rocblas_gentest.py-under-python-on-Windows.patch

```diff
@@ -0,0 +1,40 @@
+From a7245256ba4901d387e932dd1f7830da712baa56 Mon Sep 17 00:00:00 2001
+From: Scott Todd <scott.todd0@gmail.com>
+Date: Tue, 1 Jul 2025 14:34:35 -0700
+Subject: [PATCH 4/5] Run rocblas_gentest.py under `python` on Windows.
+
+---
+ clients/common/rocblas_parse_data.cpp | 11 ++++++++---
+ 1 file changed, 8 insertions(+), 3 deletions(-)
+
+diff --git a/clients/common/rocblas_parse_data.cpp b/clients/common/rocblas_parse_data.cpp
+index 83039751..399a709f 100644
+--- a/clients/common/rocblas_parse_data.cpp
++++ b/clients/common/rocblas_parse_data.cpp
+@@ -47,15 +47,20 @@ static std::string rocblas_parse_yaml(const std::string& yaml)
+         yaml_path = yaml;
+     }
+ 
+-    auto cmd = exepath + "rocblas_gentest.py --template " + exepath + "rocblas_template.yaml -o "
++#ifdef WIN32
++    // Windows prefers to only run certain file extensions as executables, so run under 'python'.
++    auto cmd = "python " + exepath + "rocblas_gentest.py --template " + exepath + "rocblas_template.yaml -o "
```

Comment:
I could hold off on this patch, or split it out into a separate PR. There will be similar patches to other `_parse_yaml` functions in other math libraries. These changes aren't load bearing for users or CI, though they would help for library developers who want to run tests locally, like me.

Reply:
Yeah I'll split into another PR or an issue on rocm-libraries.

The bug is that I have .py files set to open in VSCode on my machine, when they should be run through the python interpreter. On CI machines they use the interpreter.

![image](https://github.com/user-attachments/assets/3b9132be-2ff2-40ef-aab6-b6cd86bab817)

Reply:
Removed the patch from this PR and filed https://github.com/ROCm/rocm-libraries/issues/535.